api/internal/filter/authentication.go

```diff
@@ -90,7 +93,40 @@ func (mw *AuthenticationMiddleware) Handle(ctx droplet.Context) error {
 		return nil
 	}
 
-	if _, ok := conf.UserList[claims.Subject]; !ok {
+	userExist := false
+	switch conf.DataSourceType(claims.Audience) {
+	case conf.DataSourceTypeEtcd:
```

Comment:
In my understanding, `local` should be written into etcd through yaml configuration, and `external` is from third-party. It looks like `local` and `etcd` are still different here? In this way, users configured in yaml cannot be managed on the dashboard.